### PR TITLE
refactor: Change text to label

### PR DESCRIPTION
**Pull request by ebouJ**

### CODE REVIEW

File: src/account-info/AccountDetails.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ export default class AccountDetails extends PureComponent<Props, void> {
         userDetails={() => (
           <View style={styles.details}>
             {status && <UserStatusIndicator status={status} style={styles.statusIndicator} />}
-            <Text style={styles.info}>{email}</Text>
+            <Label style={styles.info} text={email} />
```
Comment (jainkuniya) [inline src/account-info/AccountDetails.js]: We are using `Label` for the text which needs to be translated.
https://github.com/zulip/zulip-mobile/blob/8e112271a40135a429c2fcb3eb84fccd14caa712/src/common/Label.js#L26
Email should as it is.

**Comment by ebouJ** [comment]
@jainkuniya thanks I updated the PR.

File: src/lightbox/LightboxFooter.js [hunk 2]
```diff
@@ -35,7 +36,7 @@ export default class LightboxFooter extends PureComponent<Props> {
     const { displayMessage, onOptionsPress, style } = this.props;
     return (
       <View style={[styles.wrapper, style]}>
-        <Text style={styles.text}>{displayMessage}</Text>
+        <Label style={styles.text} text={displayMessage} />
```
Comment (jainkuniya) [inline src/lightbox/LightboxFooter.js]: Also this is customises message, which is constructed using stream name, which should not be translated (actually not possible.)
But we need to translate `shared in/with you` text.
https://github.com/zulip/zulip-mobile/blob/8e112271a40135a429c2fcb3eb84fccd14caa712/src/lightbox/LightboxContainer.js#L89

File: src/common/TextAvatar.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ export default class TextAvatar extends PureComponent<Props> {
     return (
       <Touchable onPress={onPress}>
         <View style={[styles.frame, frameSize]}>
-          <Text style={[styles.text, textSize]}>{initialsFromName(name)}</Text>
+          <Label style={[styles.text, textSize]} text={initialsFromName(name)} />
```
Comment (jainkuniya) [inline src/common/TextAvatar.js]: We will be soon replacing current group icons with some good one.
#482

**Comment by ebouJ** [comment]
@jainkuniya thanks for the feedback. I will go ahead and close this PR.

**Pull request closed without merging**